### PR TITLE
Issue#25 sign in page

### LINKED ISSUE #25: Sign In Page

### PR DESCRIPTION
**Pull request by husseinTalal2**
Sign in page finished with the tests. closes #25

### CODE REVIEW

File: src/components/SignInPage/SignInSection.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  };
+
+  return (
+    <div className="d-flex align-items-center">
+      <div className="welcome-rectangle d-none d-md-flex align-items-center justify-content-center">
+        <h1>
+          WELCOME <br /> BACK
+        </h1>
+      </div>
+      <div className="signin-rectangle">
```
Comment (MustafaNawzat) [inline src/components/SignInPage/SignInSection.js:23]: use camelCase for classes naming.

File: src/containers/SignInPage/SignIn.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@import '../../styles/variables';
+.page-container {
```
Comment (MustafaNawzat) [inline src/containers/SignInPage/SignIn.scss:2]: leave one empty line between code blocks

**Comment by MustafaNawzat** [comment]
resolve all checks also

File: src/components/SignInPage/SignInSection.js (hunk 1 above)
Comment (husseinTalal2) [reply src/components/SignInPage/SignInSection.js:23]: should i use camelCase or use the dash in class names 
(signInRectangle) or (sign-in-rectangle)

Comment (husseinTalal2) [reply src/components/SignInPage/SignInSection.js:23]: @MustafaNawzat

**Pull request merged**